test/register-login.spec.js

```diff
@@ -0,0 +1,126 @@
+const {
+    expect
+} = require("chai")
+const {
+    By,
+    Builder,
+    until,
+    elementLocated,
+    error
+} = require("selenium-webdriver");
+
+describe('Test performs the testing of the login and sign up forms', () => {
+    let driver;
+    let testDataRandomizer = () => {
+        return Math.random().toString(36).replace(/[^a-z]/g, '').substr(0, 11);
+    };
+    let testNumbersRandomizer = () => {
+        return (Math.floor(Math.random() * (98765432120 - 100)) + 10000);
+    };
+
+    before(() => {
+        driver = new Builder().forBrowser('chrome').build();
+        driver.manage().window().maximize();
+
+    })
+    after(async () => {
+        await driver.sleep(9000);
+        await driver.quit();
+        driver = null;
+    })
+    it('Should test that page is loaded and the title matches the expected', async () => {
+        await driver.get('https://www.ah.nl');
+        expect(await driver.getTitle()).to.equal('Albert Heijn: boodschappen doen bij de grootste supermarkt');
+    });
+
+    it('Should test that modal is opened and buttom contains the text ', async () => {
+        await driver.wait(until.elementLocated(By.id('cookie-popup'), 10000));
+        expect(await driver.findElement(By.id('accept-cookies')).getText()).to.equal('Accepteer')
+    });
+
+    it('Should test that modal is closed after submission', async () => {
+        await driver.findElement(By.id('accept-cookies')).click();
+        expect(() => driver.findElement(By.id('cookie-popup'))()).to.throw();
```

Comment:
Please don't use assertion for error throwing while testing web interfaces.
The better way to assert whether an element is present using Selenium-Webdriver is to create a separate helper function. It can look like this:
```
const { By } = require("selenium-webdriver");

function isElementPresent(id) {
  try {
      driver.findElement(By.id(id));
      return true;
  } catch (error) {
      return false;
  }
}

module.exports = { isElementPresent }
```

Reply:
out of the scope for this task, will be implemented in the next tasks